.github/workflows/rancher-components.yaml

```diff
@@ -0,0 +1,25 @@
+name: rancher-components
+on:
+  pull_request:
```

Comment:
Please restrict this check on file changes [specifying the path](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore), as it should not require files outside of the library.

E.g.:

```yml
on:
  pull_request:
    paths:
      - 'pkg/rancher-components/**/*'
    branches:
      - master
```

Reply:
Done & tested.